src/components/DiffView/index.tsx

```diff
@@ -114,6 +115,15 @@ export class DiffViewBase extends React.Component<Props> {
 
     return (
       <div className={styles.DiffView}>
+        {diffs.length === 0 && (
+          <React.Fragment>
+            <div className={styles.header} />
+            <div className={makeClassName(styles.diff, styles.noDiffs)}>
+              {gettext('no differences')}
```

Comment:
Nit: Should we capitalize the "N", as in "No differences"?

Reply:
Yes, thanks!